content/terraform-docs-common/docs/partials/deprecation/hcp-vault-secrets.mdx

```diff
@@ -0,0 +1,8 @@
+<Highlight title="End of sale announcement">
+
+HCP Vault Secrets will no longer be available for purchase by new customers after June 30th, 2025.
```

Comment:
Maybe over thinking it, but does "by new customers" possibly imply that existing HashiCorp customers could still purchase it after? Would `HCP Vault Secrets will no longer be available for purchase after June 30th, 2025.` be more direct?

Reply:
Ooh yes! I like that! I'm going to change it everywhere!